other/web-js/client/src/utils.js

```diff
@@ -32,10 +47,54 @@ export function removeValueAtIndex(arr, idx) {
   return [...arr.slice(0, idx), ...arr.slice(idx + 1)];
 }
 
+export const createObjectId = () => {
+  // :state-start: development
+  return API_TYPE_NAME === "Data API"
+    ? EJSON.serialize(new ObjectId())
+    : new Realm.BSON.ObjectId();
```

Comment:
```suggestion
    ? EJSON.serialize(new ObjectId())
    : new ObjectId();
```

Reply:
Gonna leave this as-is to be as close to the generated versions as possible since Realm's bundled BSON package isn't guaranteed to be the full `bson` module.